_config.yml

```diff
@@ -21,12 +21,17 @@ cloudfilemanager: Tiny Drive
 enterpriseversion: TinyMCE Enterprise
 accountpage: Tiny Account
 supportname: Tiny Support
+#### Plan names
+tieroneplan: Tiny Cloud Community Plan
+tiertwoplan: Tiny Cloud Essential Plan
+tierthreeplan: Tiny Cloud Professional Plan
+enterpriseplan: Tiny Cloud Custom Plans
```

Comment:
This is incorrect, as custom plans include both cloud and self-hosted.

Reply:
updated